portfolio/src/main/java/com/google/sps/servlets/TestServlet.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/test")
+public class TestServlet extends HttpServlet
```

Comment:
Any plans for this test servlet? If you are just playing with it consider removing it since it has no function in your portfolio. You can still keep it locally.

Reply:
I wanted to update it for the next sections of the tutorial, I'll most likely change it's name when I find something to do with it so it doesn't stay as /test

Reply:
Sure, you can change it later.